### PR TITLE
Allow to completely remove some of simulcast encodings [MV-62]

### PR DESCRIPTION
**Pull request by daniel-jodlos**

### CODE REVIEW

File: src/membraneWebRTC.ts [hunk 1]
```diff
@@ -871,7 +881,14 @@ export class MembraneWebRTC {
    * ```
    */
   public enableTrackEncoding(trackId: string, encoding: TrackEncoding) {
-    let track = this.localTrackIdToTrack.get(trackId)?.track;
+    const track_context = this.localTrackIdToTrack.get(trackId)
+    let track = track_context?.track;
```
Comment (mickel8) [inline src/membraneWebRTC.ts:885]: use cammelCase

**Review by mickel8 — COMMENTED** [review]
Actually I am wondering what this gives us :thinking: I know I created issue for adding this feature but isn't it useless?

**Comment by daniel-jodlos** [comment]
> Actually I am wondering what this gives us 🤔 I know I created issue for adding this feature but isn't it useless?
This is 🔥. 🤦🏻‍♂️

Maybe some performance benefits. We're not creating transcoders for layers that we're not going to use?

**Comment by daniel-jodlos** [comment]
This turned out to be a useless feature

**Pull request closed without merging**